sparkmagic/sparkmagic/kernels/kernelmagics.py

```diff
@@ -389,7 +389,12 @@ def matplot(self, line, cell="", local_ns=None):
     def refresh_configuration(self):
         credentials = getattr(conf, 'base64_kernel_' + self.language + '_credentials')()
         (username, password, auth, url) = (credentials['username'], credentials['password'], credentials['auth'], credentials['url'])
-        self.endpoint = Endpoint(url, auth, username, password)
+        (ssl_client_cert, ssl_client_key, ssl_verify) = (credentials.get('ssl_client_cert'), credentials.get('ssl_client_key'), credentials.get('ssl_verify'),)
```

Comment:
Could we say "TLS" instead of "SSL" everywhere? It's been TLS for 20 years now :)